pkgs/servers/varnish/default.nix

```diff
@@ -51,19 +51,9 @@ let
     };
 in
 {
-  # EOL TBA
+  # EOL (LTS) TBA
   varnish60 = common {
     version = "6.0.11";
     hash = "sha256-UVkA2+tH/9MOs5BlyuAzFnmD7Pm9A6lDWic2B+HRKNs=";
   };
-  # EOL 2023-09-15
-  varnish72 = common {
-    version = "7.2.1";
-    hash = "sha256-TZN9FyCo7BnFM/ly2TA6HJiJt7/KdDeJOuXCfPIEqUA=";
-  };
-  # EOL 2024-03-15
```

Comment:
This date still matches what's on endoflife.date and hasn't passed yet. However, it will during the maintenance window for 23.11, if that's what you mean.

7.4 is out, but I seem to have missed that.